--- a/modules/videoio/src/cap_ffmpeg_impl.hpp
+++ b/modules/videoio/src/cap_ffmpeg_impl.hpp
@@ -1664,7 +1664,7 @@
 #endif

 #if LIBAVCODEC_BUILD >= CALC_FFMPEG_VERSION(52, 42, 0)
-    st->avg_frame_rate = (AVRational){frame_rate, frame_rate_base};
+    st->avg_frame_rate = AVRational{frame_rate, frame_rate_base};
 #endif
 #if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(55, 20, 0)
     st->time_base = c->time_base;
